apps/alpha-test-app/src/components/layout-demo/FlyoutDemo.vue

```diff
@@ -0,0 +1,45 @@
+<script lang="ts" setup>
+import { defineModel } from "vue";
+
+const showFlyout = defineModel<boolean>();
+
+defineProps<{ small?: boolean }>();
+</script>
+
+<template>
+  <label class="flyout-demo" :class="{ 'flyout-demo--small': small }">
+    Demo Drop Down:
+    <div class="flyout-demo__parent">
+      <input class="flyout-demo__input" width="10px" @click="showFlyout = !showFlyout" />
+      <div v-if="showFlyout" class="flyout-demo__flyout">
+        <slot></slot>
+      </div>
+    </div>
+  </label>
+</template>
+
+<style lang="scss" scoped>
+.flyout-demo {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: 100%;
+
+  &__parent {
+    position: relative;
+  }
+  &__flyout {
+    outline: 1px solid lightgrey;
+    background-color: white;
```

Comment:
```suggestion
    outline: var(--onyx-1px-in-rem) solid var(--onyx-color-base-neutral-300);
    background-color: var(--onyx-color-base-background-blank);
```

There are also many other components where this change should be applied to outline, border and background :) Maybe you can search+replace them? I didn't want to comment every occurence in this OR